test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -110,39 +130,21 @@ class NavigationBlockUtils {
 		this.requestUtils = requestUtils;
 	}
 
-	/**
-	 * Create a navigation menu
-	 *
-	 * @param {Object} menuData navigation menu post data.
-	 * @return {string} Menu content.
-	 */
-	async createNavigationMenu( menuData ) {
-		return this.requestUtils.rest( {
-			method: 'POST',
-			path: `/wp/v2/navigation/`,
-			data: {
-				status: 'publish',
-				...menuData,
-			},
-		} );
+	async selectNavigationItemOnCanvas( name ) {
```

Comment:
Prefer inlining one-liner assertions like this, makes the test a bit easier to read. `selectNavigationItemOnCanvas` doesn't really select anything here anyway 😆 .